packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -1,67 +1,43 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name rubyipmi
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.11.0
+Name: rubygem-%{gem_name}
+Version: 0.11.1
 Release: 1%{?dist}
 Summary: A ruby wrapper for ipmi command line tools that supports ipmitool and freeipmi
-Group: Development/Languages
-License: LGPLv2
+License: LGPLv2.1
 URL: https://github.com/logicminds/rubyipmi
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: ipmitool
```

Comment:
This caused a regression: https://github.com/theforeman/foreman-packaging/pull/10556